list.js

```diff
@@ -0,0 +1,94 @@
+const fs = require("fs");
+
+class List {
+  constructor(taskID, status, namaKegiatan) {
+    this.taskID = taskID;
+    this.status = status;
+    this.namaKegiatan = namaKegiatan;
+  }
+
+  static list() {
+    let list = fs.readFileSync("data.json", "utf8");
+    let data = JSON.parse(list);
+    return data;
```

Comment:
setelah JSON.parse, data sebaiknya diubah dulu dari array of object menjadi array of instance, baru kemudian di-return